services/scientist/maana-physical-quantity.md

```diff
@@ -502,56 +786,55 @@ schema { query: Query }
 | Example | How do I convert the string to a physical quantity only if its unit of measure is in a particular system of units? |
 | :--- | :--- |
 | Info | Use the optional systemRefs parameter to specify either the id or name of the required systems of units. If no systemRefs parameter is provided, the parser uses all available systemsOfUnits. |
-| Query | query { parse\(source:"400 psi",systemRefs:\[{name:"Imperial"}\]\){  name}} |
+| Query | query IMPERIAL{systemOfUnits\(systemRef:{name:"Imperial"}\){ ... SystemOfUnitsFragment}}<br>query EXAMPLE\($IMPERIAL:SystemOfUnitsInput\){ parse\(source:"400 psi",systemInputs:\[$IMPERIAL\]\){  name}} |
 | Response | { "data": { "parse": { "name": "400.0 lb/in²"}}} |
 
 | Example | How do I convert the string to a physical quantity only if it has a specific physical dimension? |
 | :--- | :--- |
-| Info | Use the optional dimensionRef parameter to specify either the id, name or formula of the required physical dimension. If no dimensionRef parameter is provided, the parser admits all physical dimensions. |
-| Query | query { parse\(source:"400 psi",dimensionRef:{name:"PRESSURE"}\){  name}} |
+| Info | Use the optional dimensionInput parameter to specify either the name or formula of the required physical dimension. If no dimensionRef parameter is provided, the parser admits all physical dimensions. |
+| Query | query PRESSURE{physicalDimension\(dimensionRef:{name:"PRESSURE"}\){ ... PhysicalDimensionFragment }}<br>query EXAMPLE($PRESSURE:PhysicalDimensionInput){ parse\(source:"400 meters",dimensionInputs:[$PRESSURE]\){  name }} |
 | Response | { "data": { "parse": { "name": "400.0 lb/in²"}}} |
 
 ### Extraction
 
 | Example | You have a list of strings, each which may contain zero or many physical quantities. How do you extract all the physical quantities from the source text? |
 | :--- | :--- |
 | Info | Use the extract query to get all the entity mentions from the text. The extract method can be used to find entity mentions in a list of string literals, or the field of a kind. The extractor will find all the physical quantity mentions in each string \(or instance of a field\) and return a list of the results. |
-| Query | { extract\(sources: \[ "the airspeed of the plane was 350 mph as it passed through 3000 meters.", "The minimum runway length is 7500 feet."\]\) {  entity {  name}}} |
-| Result | { "data": { "extract": \[ { "entity": { "name": "350.0 mi/hr"}}, { "entity": { "name": "3000 m"}}, { "entity": { "name": "7500.0 ft"}}\]}} |
+| Query | extractBatch\(sources: \[ "the airspeed of the plane was 350 mph as it passed through 3000 meters.", "The minimum runway length is 7500 feet."\]\) {  entity {  name}} |
```

Comment:
query is malformed (quotes)

Reply:
The query keyword is optional.   For consistency I have added it back in everywhere.